packages/chat-component-bindings/CHANGELOG.json

```diff
@@ -1,6 +1,33 @@
 {
   "name": "@internal/chat-component-bindings",
   "entries": [
+    {
+      "date": "Mon, 14 Aug 2023 18:28:42 GMT",
+      "tag": "@internal/chat-component-bindings_v1.7.0",
+      "version": "1.7.0",
+      "comments": {
+        "patch": [
+          {
+            "author": "beachball",
+            "package": "@internal/chat-component-bindings",
+            "comment": "Bump @internal/acs-ui-common to v1.7.0",
```

Comment:
This looks like it has duplicates?

Reply:
it has different comments for each item and they were generated automatically by beachball

Reply:
OK, so we're expecting multiple refs to the same commit then?